ci/code_checks.sh

```diff
@@ -105,7 +105,12 @@ if [[ -z "$CHECK" || "$CHECK" == "lint" ]]; then
 
     # Imports - Check formatting using isort see setup.cfg for settings
     MSG='Check import format using isort ' ; echo $MSG
-    isort --recursive --check-only pandas asv_bench
+    ISORT_CMD="isort --recursive --check-only pandas asv_bench"
+    if [[ "$GITHUB_ACTIONS" == "true" ]]; then
+         eval $ISORT_CMD | awk '{print "##[error]" $0}'
```

Comment:
```suggestion
         eval $ISORT_CMD | awk '{print "##[error]" $0}' ; $?=${PIPESTATUS[0]}
```

Not sure if you can do something like this, or similar, but this will consider the return of `awk` to fail the CI, and will always be 0.

Reply:
Thanks for this v. handy